projects/1.20-fabric/assets/chatpatches/chatpatches/lang/zh_cn.json

```
@@ -0,0 +1,136 @@
+{
+	"modmenu.descriptionTranslation.chatpatches": "一个多功能的客户端聊天模组，让聊天更加定制化，查看配置和GitHub页面了解更多信息！\n我们的Discord是https://discord.gg/3mqbvneymz！",
+
+	"text.chatpatches.title": "Chat Patches 配置",
```

```suggestion
	"text.chatpatches.title": "Chat Patches配置",
```
在1.16或更高版本下，英文字母、阿拉伯数字等非中文文本无需在前后添加空格。
具体可见我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **04.排版** 一节下。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。